frontend/src/app/main/ui/auth/register.cljs

```diff
@@ -64,6 +64,10 @@
     :email-already-exists
     (swap! form assoc-in [:errors :email]
            {:message "errors.email-already-exists"})
+    
+    :email-as-password
+    (swap! form assoc-in [:errors :password]
+       {:message "errors.email-as-password"})
```

Comment:
The `{:message...}` should be indented in the same way as previous swap!

Reply:
fixed